vagrant/lib/forklift/box_factory.rb

```diff
@@ -37,6 +38,10 @@ def load_box_file(file)
 
     def process_boxes!(boxes)
       boxes.each do |name, box|
+        next unless @box_config['operating_systems'].any? { |os| name.include?(os) }
+        next if !@box_config['fips'] && name.include?('fips')
+        next if !@box_config['devel'] && name.include?('devel')
```

Comment:
As said above: devel is not a built in box and is always in the local config. No sense in built in filtering.